### PR TITLE
Updated error codes in JSon contract for BANNO Check Withdraw

### PR DESCRIPTION
**Pull request by jordan-keenan**
Updated error codes in JSon contract for BANNO Check Withdraw

[JIRA CUS-2900](https://banno-jha.atlassian.net/browse/CUS-2900)

### CODE REVIEW

**Review by jucarson — APPROVED** [review]
Looks good!

**Review by rrobison44 — APPROVED** [review]
Looks good.  Added one comment.  Also, check the first bullet point in the "additional information" section at the bottom of the contract.  Single and double quotes are kinda weird in that section.  Probably just remove the quotes all together.

**Review by tkainz — APPROVED** [review]
Thanks Jordan!

File: withdraw-by-check/withdraw-by-check-json-contract.md [hunk 1]
```diff
@@ -94,17 +94,22 @@ Unless otherwise noted, all values will be passed as double-quote encapsulated c
...
-| 503        | Share/Loan warning found                                                                 |
-| 505        | Cross Account access attempt                                                             |
+| Error Code | Logging Error Message                                                                 | Modifier                                                                             | Additional Notes As Needed                    |
+| ---------- | ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Error Opening Letterfile [configuration file name] | : [system generated letter file read error message] ||
+|            | Error Reading Letterfile [configuration file name] | : [system generated letter file read error message] ||
+|            | CFG Letterfile Error | : No Share/Loan types defined. ||
+| 501        | Avail. Balance <= $0.00 || Target Share/Loan available balance<=$0.00 |
+| 502        | Invalid Address |||
+| 503        | Account Not Found |||
```
[the remark this answers is not in the document]
Comment (jordan-keenan) [reply withdraw-by-check/withdraw-by-check-json-contract.md]: Thanks for the suggestion, Robby. I made the change and will merge it

**Pull request merged**